.github/workflows/cut-rc.yml

```diff
@@ -0,0 +1,598 @@
+
+  create-long-running-cluster:
+    name: Create GKE long-running cluster
+    needs: [variables, cut-rc]
+    if: >-
+      github.event.inputs.dry-run != 'true' &&
+      needs.variables.outputs.rc == '1'
+    uses: ./.github/workflows/create-cluster.yml
+    with:
+      flavor: gke-default
+      name: gke-long-running-${{needs.variables.outputs.milestone}}
+      lifespan: 168h
+      args: nodes=5
+      wait: true
+
+  patch-long-running-cluster:
+    name: Patch OS4 cluster
+    needs: [variables, create-long-running-cluster]
+    runs-on: ubuntu-latest
+    env:
+      NAME: gke-long-running-${{needs.variables.outputs.milestone}}
+      TAG: ${{needs.variables.outputs.milestone}}
+      KUBECONFIG: artifacts/kubeconfig
+      INFRA_TOKEN: ${{secrets.INFRA_TOKEN}}
+    steps:
+      - name: Download infractl
+        run: |
+          curl --fail -sL https://infra.rox.systems/v1/cli/linux/amd64/upgrade \
+          | jq -r ".result.fileChunk" \
+          | base64 -d \
+          > infractl
+          chmod +x infractl
+      - name: Test readiness
+        run: |
+          CLUSTER_STATUS=$(./infractl get "$NAME" --json)
+          echo "$CLUSTER_STATUS"
+          if [ "$(echo "$CLUSTER_STATUS" | jq -r .Status)" != "2" ]; then
+            exit 1
+          fi
+      - name: Install kubectl
+        uses: azure/setup-kubectl@v2.0
+        id: install
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{needs.variables.outputs.milestone}}
+      - name: Launch central
+        env:
+          MAIN_IMAGE_TAG: ${{needs.variables.outputs.milestone}} # Release version, e.g. 3.63.0-rc.2.
+          API_ENDPOINT: localhost:8000
+          STORAGE: pvc # Backing storage
+          STORAGE_CLASS: faster # Runs on an SSD type
+          STORAGE_SIZE: 100 # 100G
+          MONITORING_SUPPORT: true # Runs monitoring
+          LOAD_BALANCER: lb
+          ROX_ADMIN_USERNAME: admin
+        run: |
+          ./deploy/k8s/central.sh
+          ROX_ADMIN_PASSWORD=$(cat deploy/k8s/central-deploy/password)
+          kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
+
+          ./deploy/k8s/sensor.sh
+
+          kubectl -n stackrox set env deploy/sensor MUTEX_WATCHDOG_TIMEOUT_SECS=0
+          kubectl -n stackrox set env deploy/sensor ROX_FAKE_KUBERNETES_WORKLOAD=long-running
+          kubectl -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+
+          kubectl -n stackrox set env deploy/central MUTEX_WATCHDOG_TIMEOUT_SECS=0
+          kubectl -n stackrox patch deploy/central -p '{"spec":{"template":{"spec":{"containers":[{"name":"central","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+          echo "Long-running GKE cluster has been patched" >> $GITHUB_STEP_SUMMARY
```

Comment:
When I created my script to create the long-running cluster I had to make some changes to have it wait for things. When copying and pasting these commands from the documentation it works, because there is time for things to finish, but if you put everything in a single scripts you need to put checks to make sure that the previous step completed. I think you will eventually replace this with my script, so it is fine.